doc/docker.md

```diff
@@ -1,16 +1,102 @@
 # Running with Docker
 
+The advantage of using docker-compose is that it is configured to run several services that the Control Panel relies on, such as the PostgreSQL database and Redis. It initializes these ready for local development purposes.
+
+## Build
+
 To build the docker image, use the following command:
+
 ```sh
 docker build -t controlpanel .
 ```
```

Comment:
(You may need to add yourself to the `docker` group (`sudo usermod -a -G docker $USER`), otherwise the command above might fail with a [permission related error](https://techoverflow.net/2017/03/01/solving-docker-permission-denied-while-trying-to-connect-to-the-docker-daemon-socket/).)

Reply:
More context: this was the case for me on a clean(ish) install of Ubuntu.

Reply:
Cool, I've added a link to the official docs about this.